mobile/test/common/jni/jni_utility_test.cc

```diff
@@ -0,0 +1,17 @@
+#include <jni.h>
+
+#include "library/common/jni/jni_utility.h"
+
+// NOLINT(namespace-envoy)
+
+// This file contains JNI implementation used by
+// `test/java/io/envoyproxy/envoymobile/jni/JniUtilityTest.java` unit tests.
+
+extern "C" JNIEXPORT jbyteArray JNICALL
+Java_io_envoyproxy_envoymobile_jni_JniUtilityTest_protoJavaByteArrayConversion(JNIEnv* env, jclass,
```

Comment:
so for every JNI method we want to test, we'll need a native test method in this file that does the similar to the function bodyhere?

Reply:
Yeah. This is how we can call those functions from Java/Kotlin.